src/mentionui.js

```diff
@@ -61,6 +70,55 @@ export default class MentionUI extends Plugin {
 		this._mentionsConfigurations = new Map();
 
 		editor.config.define( 'mention', { feeds: [] } );
+
+		this._requestFeedDebounced = debounce( ( marker, feedText ) => {
+			this._getFeed( marker, feedText )
+				.catch( error => {
+					if ( error.feedDiscarded ) {
+						// Do nothing on discarded feeds as they might come after another one was requested.
+						return;
+					}
+
+					// At this point something bad happened - most likely unreachable host or some feed callback error.
+					// So cleanup marker, remove the UI and...
+					this._hideUIAndRemoveMarker();
+
+					const notification = editor.plugins.get( Notification );
+					const t = editor.locale.t;
+
+					// ...show warning notification.
+					notification.showWarning( t( 'Could not obtain mention autocomplete feed.' ), {
```

Comment:
At this moment we don't have notifications UI at all for CKE5 itself (it's available for Letters) so this will spit out alerts AFAIR. That'd be terrible.

In fact, using global notifications (even nice) for this case seems like overkill. For now I'd log this to the console and see if we need more.

Reply:
Short notice: the [`CKFinderCommand`](https://github.com/ckeditor/ckeditor5-ckfinder/blob/a4091a1e0f321cb93ebe036cd476969e328660cc/src/ckfindercommand.js#L110) uses also notifications so it might be a case for that also (blocking notifications).